lib/ecl/ecl_sum.cpp

```diff
@@ -779,39 +779,21 @@ const char * ecl_sum_get_general_var_unit( const ecl_sum_type * ecl_sum , const
 /*****************************************************************/
 
 
-ecl_sum_type * ecl_sum_alloc_resample(const ecl_sum_type * ecl_sum, const char * ecl_case, const time_t_vector_type * _times) {
+ecl_sum_type * ecl_sum_alloc_resample(const ecl_sum_type * ecl_sum, const char * ecl_case, const time_t_vector_type * _times, const bool lower_extrapolation, const bool upper_extrapolation) {
```

Comment:
Please rename `_times` &rightarrow `times` - the leading underscore was just a f... hack.